tests/expose_test.go

```diff
@@ -39,9 +39,10 @@ func generateHelloWorldServer(vm *v1.VirtualMachine, virtClient kubecli.Kubevirt
 	defer expecter.Close()
 	Expect(err).ToNot(HaveOccurred())
 
-	serverCommand := fmt.Sprintf("screen -d -m nc -klp %d -e echo -e \"Hello World!\"\n", testPort)
+	serverCommand := fmt.Sprintf("screen -d -m nc -klp %d -e echo -e 'Hello World!'\n", testPort)
 	if protocol == "udp" {
-		serverCommand = fmt.Sprintf("screen -d -m nc -uklp %d -e echo -e \"Hello UDP World!\"\n", testPort)
+		// nc has to be in a while loop in case of UDP, since it exists after one message
+		serverCommand = fmt.Sprintf("screen -d -m sh -c \"while true\n do nc -uklp %d -e echo -e 'Hello UDP World!'\ndone\n\"\n", testPort)
```

Comment:
the udp server exited after one message (in-spite of the -lk flags), so, to support multiple node env, added it inside a "while" loop